algoliasearch/client_test.go

```diff
@@ -26,10 +26,10 @@ func TestClientOperations(t *testing.T) {
 		waitTask(t, i, res.TaskID)
 	}
 
-	t.Log("TestClientOperations: Test MoveIndex")
+	t.Log("TestClientOperations: Test RenameIndex")
 	i = c.InitIndex("TestClientOperations_copy")
 	{
-		res, err := c.MoveIndex("TestClientOperations_copy", "TestClientOperations_move")
+		res, err := c.RenameIndex("TestClientOperations_copy", "TestClientOperations_move")
```

Comment:
The `MoveIndex` method is still part of the public API. Shouldn't we keep `Test MoveIndex` and create a new one for `Test RenameIndex `?